src/Tickets/Admin/All_Tickets/List_Table.php

```diff
@@ -189,49 +190,329 @@ public function get_sortable_columns() {
+			'attendees' => $attendees_report_link,
 		];
 
-		// Setup sorting args.
-		if ( tribe_get_request_var( 'orderby' ) ) {
-			$args['orderby'] = tribe_get_request_var( 'orderby' );
+		/**
+		 * Filters the actions for the event in the All Tickets Table.
+		 *
+		 * @since TBD
+		 *
+		 * @param array $actions The actions for the event in the All Tickets Table.
+		 */
+		$actions = apply_filters( 'tec_tickets_all_tickets_table_event_actions', $actions, $event, $item );
+
+		return sprintf( '%1$s %2$s', $edit_post_link, $this->row_actions( $actions ) );
+	}
+
+	/**
+	 * Get the column start date value.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Tickets__Ticket_Object $item The current item.
+	 *
+	 * @return string
+	 */
+	public function column_start( $item ): string {
+		$date_format = tribe_get_date_format( true );
+		$datetime    = $item->start_date( false );
+
+		return $datetime->format( $date_format );
+	}
+
+	/**
+	 * Get the column end date value.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Tickets__Ticket_Object $item The current item.
+	 *
+	 * @return string
+	 */
+	public function column_end( $item ): string {
+		$date_format = tribe_get_date_format( true );
+		$datetime    = $item->end_date( false );
+
+		return $datetime->format( $date_format );
+	}
+
+	/**
+	 * Get the column days_left value.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Tickets__Ticket_Object $item The current item.
+	 *
+	 * @return string
+	 */
+	public function column_days_left( $item ): string {
+		$datetime = $item->end_date( false );
+		$now      = new DateTime();
+		$interval = $now->diff( $datetime );
+
+		if ( $interval->invert ) {
+			return '-';
 		}
 
-		if ( tribe_get_request_var( 'order' ) ) {
-			$args['order'] = tribe_get_request_var( 'order' );
+		return $interval->days;
+	}
+
+	/**
+	 * Get the column price value.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Tickets__Ticket_Object $item The current item.
+	 *
+	 * @return string
+	 */
+	public function column_price( $item ): string {
+		return tribe_format_currency( number_format( $item->price, 2 ), $item->ID );
+	}
+
+	/**
+	 * Get the column sold value.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Tickets__Ticket_Object $item The current item.
+	 *
+	 * @return string
+	 */
+	public function column_sold( $item ): string {
+		return $item->qty_sold();
+	}
+
+	/**
+	 * Get the column remaining value.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Tickets__Ticket_Object $item The current item.
+	 *
+	 * @return string
+	 */
+	public function column_remaining( $item ): string {
+		$available = $item->available();
+		return $available < 0 ? '-' : $available;
+	}
+
+	/**
+	 * Get the column sales value.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Tickets__Ticket_Object $item The current item.
+	 *
+	 * @return string
+	 */
+	public function column_sales( $item ): string {
+		return tribe_format_currency( number_format( $item->qty_sold() * $item->price, 2 ), $item->ID );
```

Comment:
This one has a logic issue. The sales can't always be quantity sold times price since the price could have changed or we could have sales. This needs to be calculated differently.

Reply:
Yes. We will need to discuss this with @bordoni